decentraland/unity-explorer
# fix: Accessing the passport on various players can freeze the UI

## What does this PR change?
Fix #1676 

During the playtest accessing random players passports caused a freeze of the whole UI.

It could be related to how we cancel the wearables requests when open the passport and then we quickly close it before the requests have finished, so I made all cancellation token sources to be always cancelled and disposed when the passport closes. In this way we ensure that no request is running while the passport is closed.

I also applied some other small fixes.

## How to test the changes?
1. Launch the explorer.
2. Go to a place with several avatars.
3. Click on each avatar to open his passport.
4. Try to do the step 3 changing to another avatars very quickly before the previous one has been loaded.
5. And the end you should check that the app is not freezed in any moment doing this.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
